modules/check_in/spec/request/v2/patient_check_ins_request_spec.rb

```diff
@@ -74,6 +76,23 @@
         expect(response.status).to eq(200)
         expect(JSON.parse(response.body)).to eq(resp)
       end
+
+      context 'with service_refactor feature flag on' do
+        it 'returns a success response' do
+          allow(Flipper).to receive(:enabled?)
+            .with('check_in_experience_services_refactor').and_return(true)
+          allow_any_instance_of(CheckIn::V2::Session).to receive(:redis_session_prefix).and_return('check_in_lorota_v2')
+          allow_any_instance_of(CheckIn::V2::Session).to receive(:jwt).and_return('jwt-123-1bc')
+          allow_any_instance_of(::V2::Lorota::Service).to receive(:check_in_data).and_return(resp)
+
+          Rails.cache.write(key, 'jwt-123-1bc', namespace: 'check-in-lorota-v2-cache')
```

Comment:
Same as above, as these statements are for test setup, can we please have these defined under before block?

Reply:
addressed in 6ea726dd366a3daf29ab012fc161ed7dd6a090f7